packages/dd-trace/src/telemetry/index.js

```diff
@@ -126,10 +136,12 @@ function stop () {
   }
   clearInterval(interval)
   process.removeListener('beforeExit', onBeforeExit)
+
+  telemetryStopChannel.publish(getTelemetryData())
```

Comment:
This will be wrongly emitted if telemetry is enabled, then disabled, then enabled again without a config file, then disabled. To avoid this, `config` should be set to `undefined` in `stop`.

Reply:
fixed